HW6/HW6.py

```diff
@@ -0,0 +1,33 @@
+# ---------------------------------------------------------------------------
+# 1 TASK
+
+list_even = []
+list_odd = []
+list_else = []
+for i in range(1, 11):
+    if i % 2 == 0:
+        list_even.append(i)
+    elif i % 3 == 0:
+        list_odd.append(i)
+    else:
+        list_else.append(i)
+
+print("Here is the list of even numbers that are divisible by 2 ", list_even)
+print("Here is the list of odd numbers that are divisible by 3: {}".format(list_odd))
+print(f"Here is the list of all left numbers: {list_else}")
+
+# ---------------------------------------------------------------------------
+# 2 TASK
+
```

Comment:
look at this implementation

def check_login():
    login = "First"
    while True:
        input_login = input("Enter your login: ")
        if input_login == login:
            print("Nice, you passed ")
            break
        else:
            print("Try again ")